api/db.py

```diff
@@ -93,6 +98,9 @@ async def update(self, obj):
         if obj.id is None:
             raise ValueError("Cannot update object with no id")
         col = self._get_collection(obj.__class__)
+        if obj.__class__ == Node:
+            obj.created = obj.id.generation_time
```

Comment:
The `created` time should be set when the object is created using the default value constructor.  So I don't think we need to update it here, only the `updated` time needs to. It would also be better to have an `update()` method in the base class of the models so we don't need to check the type of the object in `Database`.